composer.json

```diff
@@ -30,7 +30,7 @@
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "~2.1",
-        "phpunit/phpunit": "5.*"
+        "phpunit/phpunit": "<6.0"
```

Comment:
Would the tests work with PHPUnit 3.7? 😼
I think not. So something like ``>=4.0 <6.0`` would be _more_ correct.